managed/src/test/java/com/yugabyte/yw/common/TemplateManagerTest.java

```diff
@@ -204,7 +205,7 @@ public void testTemplateCommandError() {
     expectedCommand.add("prometheus");
     when(shellProcessHandler.run(eq(expectedCommand), eq(new HashMap<>()), anyString()))
         .thenReturn(ShellResponse.create(1, "foobar"));
-    expectedException.expect(RuntimeException.class);
+    expectedException.expect(YWServiceException.class);
     expectedException.expectMessage("YBCloud command instance (template) failed to execute.");
     templateManager.createProvisionTemplate(accessKey, true, true, true, 9300, "prometheus");
     assertAccessKeyInfo(accessKey, false, false, true);
```

Comment:
Which line are we expecting exception from?
I think the test was broken in that this last line will never be hit. 
Confirm and remove this check if thats the case.